product_docs/docs/biganimal/release/overview/02_high_availability.mdx

```diff
@@ -61,6 +61,8 @@ Extreme-high-availability clusters are only Oracle compatible.
 
 Extreme-high-availability clusters are configured according to *data groups*. EDB Postgres Distributed (PGD) clusters create a PGD global group, which contains one or two data groups. Your data groups can be made up of a combination of data nodes and witness nodes. One of these data nodes is the leader at any given time, while the rest are shadow nodes. 
 
+PGD Proxy routes all application traffic to the leader node, which acts as the principal write target to reduce the potential for data conflicts. PGD Proxy leverages a distributed consensus model to determine availability of the data nodes in the cluster. On failure or unavailability of the leader, PGD Proxy elects a new leader and redirects application traffic. Together with the core capabilities of EDB Postgres Distributed, this mechanism of routing application traffic to the leader node enables fast failover and switchover.
```

Comment:
This being the first reference to PGD Proxy in this documentation, might be nice to see a link over to /pgd/latest/routing/proxy

Reply:
good point. Do you think it is ok to make that change to the release branch? Or I should I not worry about it for this build?